Wireframe/css/style.css

```diff
@@ -0,0 +1,160 @@
+@import url('https://fonts.googleapis.com/css2?family=Arsenal&display=swap');
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+}
+
+:root {
+    --dark-blue: #071221;
+    --white: #ffff;
+
+}
+
+body {
+    margin: 0;
+    min-height: 100vh;
+
+}
+
+body,
+button {
+    font-family: 'Arsenal', sans-serif;
+    color: var(--grey-dark);
+    font-size: 1rem;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    margin-top: 0;
+    margin-bottom: 0;
+}
+
+p {
+    margin-top: 0;
+    margin-bottom: 0;
+}
+
+a {
```

Comment:
How will your user know what links are clickable? Have you planned this interaction?